agent/php_observer.c

```diff
@@ -75,13 +76,58 @@
 static zend_observer_fcall_handlers nr_php_fcall_register_handlers(
     zend_execute_data* execute_data) {
   zend_observer_fcall_handlers handlers = {NULL, NULL};
+  zend_string *func_name = NULL, *scope_name = NULL;
+  nruserfn_t* wr;
   if (NULL == execute_data) {
     return handlers;
   }
   if ((NULL == execute_data->func)
       || (ZEND_INTERNAL_FUNCTION == execute_data->func->type)) {
     return handlers;
   }
+
+  if (nrunlikely(NR_PHP_PROCESS_GLOBALS(special_flags).show_executes)) {
+    nr_php_show_exec("observe", execute_data, NULL);
+  }
+
+  if (OP_ARRAY_IS_A_FILE(NR_OP_ARRAY)) {
+    /*
+     * Let's get the framework info.
+     */
+    nr_php_execute_file(NR_OP_ARRAY, execute_data, NULL TSRMLS_CC);
+    return handlers;
+  }
+
+  if (OP_ARRAY_IS_A_METHOD(NR_OP_ARRAY)) {
+    scope_name = execute_data->func->op_array.scope->name;
+    func_name = execute_data->func->op_array.function_name;
+#if 0
```

Comment:
Are these just for debugging?  Maybe if we want to leave some troubleshooting code around have a define like DEBUG_FCALL_HANDLERS at the top of the file which is false by default instead of `#if 0`?

Reply:
This code has been cleaned up and simplified in 09eca036c48b27d7d27a20137b268d9543a59676. There is no need for any of this debugging code.

Reply:
Should we output the hash value in show executes?  Maybe just for PHP 8+?